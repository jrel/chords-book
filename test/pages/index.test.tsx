import React from 'react'
import { render } from '../testUtils'
import Index from '../../pages/index'

describe('Home page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Index categories={[]} />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
