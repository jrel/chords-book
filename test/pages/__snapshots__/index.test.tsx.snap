// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-root-1"
  >
    <ul
      class="MuiList-root MuiList-padding"
    />
  </div>
</DocumentFragment>
`;
